### PR TITLE
Add Aave v3 EtherFi Ethereum addresses

### PR DESCRIPTION
**Pull request by kartojal**
### Description
Add EtherFi/Stables V3 instance addresses and helpers as requested by [[ARFC] Deploy an Etherfi/Stablecoin Aave v3 Instance](https://governance.aave.com/t/arfc-deploy-an-etherfi-stablecoin-aave-v3-instance/18440) proposal.

For reference you can check the following Catapulta deployment report:
https://catapulta.sh/report/719c68e7-9829-455d-a580-b5c93b25812a

### CODE REVIEW

File: scripts/configs/pools/ethereum.ts [hunk 1]
```diff
@@ -106,3 +106,23 @@ export const lidoEthereumMainnetProtoV3Pool: PoolConfig = {
...
+    WITHDRAW_SWAP_ADAPTER: '0x850347E0cF64fd342A3404c1c5DA21Aa0A46c5c6',
+    UI_INCENTIVE_DATA_PROVIDER: '0x8Ed2Ac8a525BA1CfF1b4253de62Ef8466c5C8311',
+    UI_POOL_DATA_PROVIDER: '0xdbbB1C304FA7feDeF5855ab993d34254BF2576A6',
+    WALLET_BALANCE_PROVIDER: '0x1C438F26Dc3751f70fCBf63493d94EaA1F31c8fC',
+    WETH_GATEWAY: '0xAB911dFB2bB9e264EE836F30D3367618f8Aef965',
+    CONFIG_ENGINE: '0x482548b76D4C577A0b5cb2A1f97EFB2893E957ed',
+    STATIC_A_TOKEN_FACTORY: '0x1Ff525426800279843B71C0F818594DeCdC3b522',
+    // CAPS_PLUS_RISK_STEWARD: '',
+    // RISK_STEWARD: '',
+  },
```
Comment (brotherlymite) [inline scripts/configs/pools/ethereum.ts:127]: I would also add the collector and token impl addresses as initial addresses

File: src/AaveV3EthereumEtherFi.sol [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  address internal constant WITHDRAW_SWAP_ADAPTER = 0x850347E0cF64fd342A3404c1c5DA21Aa0A46c5c6;
+
+  // https://etherscan.io/address/0x8Ed2Ac8a525BA1CfF1b4253de62Ef8466c5C8311
+  address internal constant UI_INCENTIVE_DATA_PROVIDER = 0x8Ed2Ac8a525BA1CfF1b4253de62Ef8466c5C8311;
+
+  // https://etherscan.io/address/0xdbbB1C304FA7feDeF5855ab993d34254BF2576A6
+  address internal constant UI_POOL_DATA_PROVIDER = 0xdbbB1C304FA7feDeF5855ab993d34254BF2576A6;
+
+  // https://etherscan.io/address/0x1C438F26Dc3751f70fCBf63493d94EaA1F31c8fC
+  address internal constant WALLET_BALANCE_PROVIDER = 0x1C438F26Dc3751f70fCBf63493d94EaA1F31c8fC;
```
Comment (brotherlymite) [inline src/AaveV3EthereumEtherFi.sol:68]: I would use the same address as of ETH v3 instance

File: scripts/configs/pools/ethereum.ts (hunk 1 above)
Comment (kartojal) [reply scripts/configs/pools/ethereum.ts:127]: added!

File: src/AaveV3EthereumEtherFi.sol (hunk 2 above)
Comment (kartojal) [reply src/AaveV3EthereumEtherFi.sol]: added!

File: src/AaveV3EthereumEtherFi.sol [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    0x8831a5EEB5e10D2E5148Ca91ee9E4716a00c2CB2;
+
+  // https://etherscan.io/address/0xB04427eFdd15b0EC233400d2F7f7E4fd0291C285
+  address internal constant SWAP_COLLATERAL_ADAPTER = 0xB04427eFdd15b0EC233400d2F7f7E4fd0291C285;
+
+  // https://etherscan.io/address/0x850347E0cF64fd342A3404c1c5DA21Aa0A46c5c6
+  address internal constant WITHDRAW_SWAP_ADAPTER = 0x850347E0cF64fd342A3404c1c5DA21Aa0A46c5c6;
+
+  // https://etherscan.io/address/0x8Ed2Ac8a525BA1CfF1b4253de62Ef8466c5C8311
+  address internal constant UI_INCENTIVE_DATA_PROVIDER = 0x8Ed2Ac8a525BA1CfF1b4253de62Ef8466c5C8311;
```
[the remark this answers is not in the document]
Comment (kartojal) [reply src/AaveV3EthereumEtherFi.sol]: added!

**Review by sendra — APPROVED** [review]

**Pull request merged**